wiki/backup.md

```diff
@@ -36,13 +36,13 @@ File History is a tool which automatically makes backups of your files. You can
 
 You can use System Image Backup to back up your system image. A system image is basically an image of the current state of the operating system stored in a file. You can use these system images to revert the system to a previous state.
 
-![System Image Backup option](img/backup/system-image-backup.png)
+![System Image Backup option](./img/backup/system-image-backup.png)
 
 - Search "File History" and then select it.
 - On the Control Panel sidebar, click on "System Image Backup".
-![](img/backup/create-a-system-image.png)
+![](./img/backup/create-a-system-image.png)
```

Comment:
Not just this one but, what is the purpose of this change? It's not mentioned in the PR body.

Reply:
The message of the commit that adds it describes why.

Reply:
![Uploading Screenshot_20230527_225543_Chrome.jpg…]()

Reply:
Okay so can you point that out? It's all reviewed as one PR and it's difficult to drill down into each commiit when there are many

Reply:
Alright done, I'd suggest adding a template for issues and PRs to make stuff like this easier to write.

Reply:
You lost me, what part of them breaks if one is selfhosting the site? I've built this thing before and it's worked the same, is there any specific environment that it breaks under?

Reply:
Thanks

Reply:
> You lost me, what part of them breaks if one is selfhosting the site? I've built this thing before and it's worked the same, is there any specific environment that it breaks under?

Basically every image doesn't load and results in a 404 error on my end, adding `./` fixed it. I thought this was an issue with the vuepress tips.

Reply:
Oh sure, let me have a quick look 😅 might be the tips indeed

Reply:
Weird tho, last time I tried it works on the deploy preview built on Netlify. Might be a localhost issue? I was able to reproduce that issue on `main` before iirc.

Reply:
Nah you're right it happens when running locally, I think Vuepress made a change regarding asset handling, so I'll take the changes for this. ty